site/gatsby-site/server/emails/index.ts

```diff
@@ -35,6 +36,15 @@ export const replacePlaceholdersWithAllowedKeys = (template: string, data: { [ke
     });
 }
 
+let bulkLimiter = new RateLimiter({
```

Comment:
Please add a comment with a link to the MailerSend documentation to know why we need to limit to 10 calls per second

Reply:
Added